Binary Search Trees/Binary Search Tree Implementation Complete.py

```diff
@@ -0,0 +1,202 @@
+class Node:
+    def __init__(self,value):
+        self.value = value
+        self.left = None
+        self.right = None
+    
+    def insert(self, value):
+        if self.value == value:
+            return
+        if self.value > value:
+            if self.left:
+                return self.left.insert(value)
+            else:
+                self.left = Node(value)
+        if self.value < value:
+            if self.right:
+                return self.right.insert(value)
+            else:
+                self.right = Node(value)
+    
+    def find(self, value):
+        found = False
+        if self.value == value: #check to see if the current node's value matches the search value. NOTE - initiation self == root.
+            found = True
+        elif self.value == None:
+            found = False
+        elif self.value < value: #if the value is greater, check the right side of the tree if right side is there.
+            if self.right:
+                return self.right.find(value)
+            else:
+                found = False
+        else:
+            if self.left: #if the value is less, check the left side of the tree if left side is there.
+                return self.left.find(value)
+            else:
+                Found = True
+        if found:
+            print("Value {} in the tree".format(value))
```

Comment:
Print statement starting with uppercase, whereas other print statements start with lowercase letters. Follow one and stick with it throughout the code base.